Exiled.CustomItems/Events/PlayerHandler.cs

```diff
@@ -0,0 +1,55 @@
+// -----------------------------------------------------------------------
+// <copyright file="PlayerHandler.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.CustomItems
+{
+    using Exiled.API.Extensions;
+    using Exiled.API.Features;
+    using Exiled.CustomItems.API.Features;
+    using Exiled.Events.EventArgs;
+
+    /// <summary>
+    /// Handles Player events for the CustomItem API.
+    /// </summary>
+    internal sealed class PlayerHandler
+    {
+        public void OnChangingRole(ChangingRoleEventArgs ev)
+        {
+            switch (ev.NewRole == RoleType.Spectator)
```

Comment:
Why a switch instead of an if?

Reply:
Uhm. no real reason, does it particularly matter?